Rebbi add blogpost

reebxu459 opened the pull request.

jilly477 committed:
attempted to create backend for opportunities page. pushed to inspect prisma database error

xsharonhe committed:
Add table name changes

xsharonhe committed:
Commit changes

jilly477 committed:
pretty formatting

jilly477 committed:
Merge branch 'main' of https://github.com/PuMProfessionals/pump-web-v2 into add-opportunities-backend

jilly477 committed:
fix package-lock.json merge conflicts

jilly477 committed:
delete package-lock,json

xsharonhe committed:
Temporary commit

xsharonhe committed:
Package-lock.json fix

jilly477 committed:
pulled from main and fixed merge conflicts

jilly477 committed:
merged with the current branch

jilly477 committed:
added package-lock.json and deleted the extra baycrest posting

jilly477 committed:
updated search feature of opportunity.js

jilly477 committed:
Merge branch 'main' of https://github.com/PuMProfessionals/pump-web-v2 into add-opportunities-backend

jilly477 committed:
fixed up opportunity backend after recloning

jilly477 committed:
finished up search functionality for opportunities

reebxu459 committed:
add blogpost file

reebxu459 committed:
add Gut Health file

reebxu459 committed:
add second blogpost file

reebxu459 committed:
add second blogpost

hirundinidae commented on _digest/october-2020/an-effective-guide-to-bullet-journalling.mdx:
```diff
@@ -0,0 +1,43 @@
+    <p>In recent years, an abrupt boom in studygrams on social media has taken over the productivity world. Although aesthetic note taking, stationery, and handwriting go through on-and-off phases of popularity, one key aspect to an aesthetically productive life has always been trending- bullet journaling.&nbsp;</p>
+    <p>Those who are new to bullet journaling struggle to devise how to properly use the technique to organize their schedule and to-do lists in order to reap optimal results of a simultaneous creative outlet and effective organization system. But, knowing the right basics, bullet journaling can help one optimize their time to lead both a productive and healthy life. Today, I will be sharing my g ...
+    <p>Before starting to bullet journal, it is crucial for one to find a bullet journal that suits their style and their needs. Not only will it help with creating beautiful bullet journal spreads, but it can also aid with an effective planning system, as you should be comfortable with using the book for long periods of time. The following is a short list of bullet journaling notebooks that I pe ...
+    <ol><li><a href="https://www.amazon.com/Dotted-Bullet-Grid-Journal-Notebook/dp/B07FD7FSW3"><strong>Plouma Dotted Grid Notebook</strong></a><strong> ($7.79 CAD)</strong></li></ol>
```
Take a look at how the other blog posts add links and follow their example for all the links in this blog post
```
<a href="https://learningcommons.ubc.ca/mental-health-and-balancing-student-work-life/" target="_blank" rel="noopener noreferrer">Mental Health and Balancing Student Work Life</a>
``` 
from https://raw.githubusercontent.com/PuMProfessionals/pump-web-v2/main/_digest/october-2020/incorporate-social-life-in-student-life.mdx 
 (add the `target=` and `rel=` attributes)

hirundinidae commented on _digest/october-2020/an-effective-guide-to-bullet-journalling.mdx:
```diff
@@ -0,0 +1,43 @@
+    <p>Now, to jump into the basics of bullet journaling: I recommend starting off with discovering the different spreads many students use to organize their life. To this day, although I have outgrown a few, I still use many due to their effectiveness. This takes me to my next point: Please do not hesitate to discontinue spreads if they are overwhelming or hard to keep up with. I see a lot of pe ...
+    <p><strong>Student&nbsp; Layouts</strong></p>
+    <p><strong>Semester layout</strong></p>
+    <figure class="wp-block-image"><img src="https://lh4.googleusercontent.com/Q8OxSB7i2MZBQfmEm-AyeyIg1k24XnzjC17jqrXH6_MDIlLqxFDRq1a6DzldWBEwPn0-RgbPDqi3BnIhitzLkCNjcZ_r2q3CUrzEIUqyAz3aU0f3Hu7JTL5Le8AH4vgPSoqMsEWp" alt=""></figure>
```
Remove the attributes that are `class=wp-*` and I think the images should be downloaded and put into the public/blog folder (sorry, forgot to mention this in our meeting)

hirundinidae commented on _digest/october-2020/an-effective-guide-to-bullet-journalling.mdx:
```diff
@@ -0,0 +1,43 @@
+    <p>Before starting to bullet journal, it is crucial for one to find a bullet journal that suits their style and their needs. Not only will it help with creating beautiful bullet journal spreads, but it can also aid with an effective planning system, as you should be comfortable with using the book for long periods of time. The following is a short list of bullet journaling notebooks that I pe ...
+    <ol><li><a href="https://www.amazon.com/Dotted-Bullet-Grid-Journal-Notebook/dp/B07FD7FSW3"><strong>Plouma Dotted Grid Notebook</strong></a><strong> ($7.79 CAD)</strong></li></ol>
+    <p>I recommend this book to beginners who are not sure if bullet journaling is the type of planner for them. It is relatively inexpensive and is very effective to draw, colour, and write on.</p>
+    <ol start="2"><li><a href="https://www.amazon.ca/Kesoto-Eco-Friendly-Natural-Hardcover-Notebook/dp/B07D564SLL/ref=pd_lpo_229_t_1/135-8501544-8737535?_encoding=UTF8&amp;pd_rd_i=B07D564SLL&amp;pd_rd_r=64a6e8f6-6322-4632-af0f-f19ddb187225&amp;pd_rd_w=VYUfZ&amp;pd_rd_wg=cdBzC&amp;pf_rd_p=256a14b6-93bc-4bcd-9f68-aea60d2878b9&amp;pf_rd_r=37SPCK43990P40CXBHPB&amp;psc=1&amp;refRID=37SPCK43990P40CXBHP ...
+    <p>I recommend this book to people who are semi-dedicated to bullet journaling, or/and are concerned about the environment. It can be used with a wide range of mediums and is also relatively cheap compared to other products on the market that are eco friendly!&nbsp;</p>
+    <ol start="3"><li><a href="https://www.amazon.ca/Leuchtturm1917-Notebooks-Hardcover-Medium-Powder/dp/B07ZXJDY1Y/ref=sr_1_3_sspa?dchild=1&amp;keywords=Leuchtturm1917+Medium+A5+Notebook&amp;qid=1601422379&amp;s=office&amp;sr=1-3-spons&amp;psc=1&amp;spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEyMjZCOEZBMVZEMEFFJmVuY3J5cHRlZElkPUEwNzEyMDI5MjRESjNGV0ZTS0FaQyZlbmNyeXB0ZWRBZElkPUEwNDA1NjYxMTNPS0FRVTlOSVZOVCZ3aWR ...
```
instead of `<ol start="number"`, change the tags to use 1 `<ol>` for each list (look at the current blog post to see where the list starts/ends) and add paragraph tags after the numbered list item. This is similar to this blog post Bhumika did. https://raw.githubusercontent.com/PuMProfessionals/pump-web-v2/main/_digest/october-2020/incorporate-social-life-in-student-life.mdx
```
<ol> 
  <li>
  <p>
  <li> 
  <p> 
  ... 
</ol> 
```

hirundinidae commented on _digest/october-2020/an-effective-guide-to-bullet-journalling.mdx:
```diff
@@ -0,0 +1,43 @@
+    <figure class="wp-block-image is-resized"><img loading="lazy" src="https://lh3.googleusercontent.com/EbURGP8XnG_Er9xqB3AniHaKsOjsU_WIRfO_LiSjH5BaGdcdNgri96WiHTGW0ZJhuHDB99iOGyY07FzARE7SFYWfNheZ_1SDtRe7LUB1Ht4Neu2tn0vmTF00-WqKOBbljYUs3zOj" alt="" width="492" height="655"></figure>
+    <p><strong>Self Care Spreads</strong></p>
+    <p><strong>Quotes pages</strong></p>
+    <figure class="wp-block-image is-resized"><img loading="lazy" src="https://lh6.googleusercontent.com/uppgyRSlI-E-Xvgqh-VWPGQmdQgxvhQQt7Bt1BysLRDZwplfV-jbkVPTGsxJAbDeZ9GaVYs1wWCmlgKBz7NmSedz_DN3p8-4BeuQwShNeDx3dqXLq4cgarnjmiLPsn0AtBZlWr_R" alt="" width="501" height="667"></figure>
```
Add appropriate alt text to the image tags

bhumika-gupta commented:
One thing I would say is, make sure you're not simply copying and pasting the code from the old site, and that you're actually going through it to remove the classes and other properties within the tags. Remember to follow the guidelines on the README.md of this repo (scroll near the bottom https://github.com/PuMProfessionals/pump-web-v2). Look at commit history for previous blog articles added

jilly477 committed:
implemented minor changes from last review. Updated the search filter function and cleaned away some comments

jilly477 committed:
Merge branch 'main' of https://github.com/PuMProfessionals/pump-web-v2 into add-opportunities-backend

xsharonhe committed:
Upgrade packages

xsharonhe committed:
Update package-lock.json

jilly477 committed:
updated seed.js and cache.js to read string input and transfer it over to boolean meaning

jilly477 committed:
Merge branch 'main' of https://github.com/PuMProfessionals/pump-web-v2 into add-opportunities-backend

hirundinidae committed:
Add author component (#48)

* Add author components

* Format files

* Add authors as props and fix margins

* Add proptypes and comments

* Make suggested changes

set default profile picture to tie dye background
delete previous profile pictures svg
change to ===

* Format files

jilly477 committed:
final fix, changed the archive command in seed.js and cache.js

jilly477 committed:
Merge branch 'main' of https://github.com/PuMProfessionals/pump-web-v2 into add-opportunities-backend

jilly477 committed:
Merge pull request #43 from PuMProfessionals/add-opportunities-backend

Add opportunities backend

hirundinidae committed:
Add opportunities (#45)

* Add opportunities

* Remove finished files

* Format

* Add placeholder values and archived/published key

* Move files to _direct

xsharonhe committed:
Update README.md

xsharonhe committed:
Update README.md

alyssasuiyiguo committed:
Added two PuMP digest articles (#32)

* Added article frontmatter

* Finished morning routine article

* Added PharmD article

* Added images to morning blog article

* Added image to PharmD article

* Removed line spaces from blog article

* Removed line spaces from second blog article

xsharonhe committed:
Fix blog markdowns (#47)

* Fix blog markdowns

* Prettier changes

* Fix db changes

* Prettier fixes

* Image wrapper additions

* Add remaining board members

* Add photos to website

* Add author to blog

jilly477 committed:
multi select add-on for backend

xsharonhe committed:
Add seo content (#60)

* Release fixes

* Initial blog SEO changes with favicons and og tags and descriptions for some pages

* Add SEO and arrowLink to individual pages

* Add descriptions

* Fix prettier

* Prettier and unterminated tags

Co-authored-by: hirundinidae <diana4lin@gmail.com>

jilly477 committed:
created funcitonal v1 of backend dual filtering with search bar and tags for blog

jilly477 committed:
finished backend filtering for blogs

jilly477 committed:
finished backend filtering for opportunities

jilly477 committed:
finished format search page for blogs and opportunities

jilly477 committed:
merged with main and resolved merge conflict on the blog.jsx file

jilly477 committed:
fixed lint issue regarding an unused props variable in multiselector components

jilly477 committed:
added package-lock.json and fixed changes suggested by review

hirundinidae committed:
Add blog card (#62)

* Add author structure

* Adjust widths for each section

* Adjust widths to use Author in a div

* Add desktop layout

* Add background colors

* Add tags and fix margins

* Add styling for mobile

* Use Link tag in Title

* Add BlogSection to blog page

* Format

* Add props to blogcard

* Fix alignment, add hover

* Add blog posts

* Format

* Add tag component

* Format

bhumika-gupta committed:
Add opportunities task (#51)

* Remove top property for BackArrow media query

* Change width of TitleSection media query

* Change left positioning of BackArrow media query

* Change top positioning of BackArrow media query

* Add york uni spark opp

* Add ucla nanoscience opp

* Fix link to image

* Add volunteering scarborough health network opp

* Add chartwell retirement opp

* Add volunteering providence healthcare opp

* Add discover hopkins opp

* Add woodgreen foundation opp

* Add kensington health opp

* Add circle of care opp

* Add belmont senior home opp

* Add st.john ambulance opp

* Add gap medics opp

* Add projects abroad opp

jilly477 committed:
merged with main and fixed merge conflicts

jilly477 committed:
updated formatting for blog page to look good together with the blog cards

jilly477 committed:
fixed minor css details within the blog interface

jilly477 committed:
Merge pull request #61 from PuMProfessionals/add-filter-interface

Add search filtering within opportunities and blog pages

bhumika-gupta committed:
Adjust Title component (#50)

* Remove top property for BackArrow media query

* Change width of TitleSection media query

* Change left positioning of BackArrow media query

* Change top positioning of BackArrow media query

* Fix top positioning for BackArrow media query

* Change positioning of Back Arrow

* Remove  commented out code

* Change positioning of back arrow

* Change positioning of back arrow on normal screen

* Add new div for arrow and title

* Fix small edit

* Change width of TopSection

xsharonhe committed:
Extra formatting (#68)

* Fixing blog and board sections

* Fix package-lock

* Add multiselector pacakge

* Changing blogs summary

* Beginning carousel changes

* Add all events

* Prettieer fixes

* Finish formatting opportunities slug

* Formatting

xsharonhe committed:
Small title fix

bhumika-gupta committed:
Merge branch 'rebbi-add-blogpost' of https://github.com/PuMProfessionals/pump-web-v2 into rebbi-add-blogpost

bhumika-gupta committed:
Fix bullet journaling blog post

bhumika-gupta committed:
Add images for bullet journaling blog post

bhumika-gupta committed:
Fix spaces for journaling blog post

bhumika-gupta commented:
@xsharonhe @hirundinidae I just fixed the blog post just so we're done with these articles (and it renders fine when I checked it locally). Also, can someone please explain why so many other commits showed up? I keep thinking I'm doing something wrong every time and I can never find a proper answer online.

hirundinidae reviewed (approved):
Checked the blog posts as well, everything renders. Not sure about commits though, @/Sharon

xsharonhe commented:
Commits are because this PR is very old and pulls all past commits. This is fine
